nullaway/src/test/java/com/uber/nullaway/NullAwayCustomLibraryModelsTests.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2022 Uber Technologies, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.uber.nullaway;
+
+import com.google.errorprone.CompilationTestHelper;
+import com.uber.nullaway.testlibrarymodels.TestLibraryModels;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class NullAwayCustomLibraryModelsTests extends NullAwayTestsBase {
+
+  private CompilationTestHelper makeLibraryModelsTestHelperWithArgs(List<String> args) {
+    // Adding directly to args will throw an UnsupportedOperationException. Throughout all tests in
+    // NullAway, list of args are created via Arrays.asList which the returned list does not support
+    // addAll. To comply with the rest of the tests, we created tests arguments with the same
```

Comment:
This comment is a bit cumbersome to parse, at least to me, I would rewrite it as:

> Adding directly to args will throw an UnsupportedOperationException, since that list is created by calling Arrays.asList (for consistency with the rest of NullAway's test cases), which produces a list which doesn't support add/addAll. Because of this, before we add our additional arguments, we must first copy the list into a mutable ArrayList.

Reply:
15cef2fd8938ef3388b0f80870437fce5a235c7c